include/bluetooth/audio/mcc.h

```diff
@@ -923,9 +923,9 @@ int bt_mcc_otc_read_current_group_object(struct bt_conn *conn);
 int bt_mcc_otc_read_parent_group_object(struct bt_conn *conn);
 
 #if defined(CONFIG_BT_MCC_SHELL)
-struct bt_otc_instance_t *bt_mcc_otc_inst(void);
+struct bt_ots_client *bt_mcc_otc_inst(void);
```

Comment:
```suggestion
struct bt_ots_client *bt_mcc_ots_client_inst(void);
```

Reply:
Can be done in a separate PR, as it is a bit unrelated to this PR

Reply:
Will most likely be fixed when the server and client instances are unified.

Reply:
> Will most likely be fixed when the server and client instances are unified.

Do we have an issue for that?

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/42642